src/diffusers/pipelines/controlnet/pipeline_controlnet_inpaint.py

```diff
@@ -868,8 +868,8 @@ def prepare_latents(
             # if pure noise then scale the initial latents by the  Scheduler's init sigma
             latents = latents * self.scheduler.init_noise_sigma if is_strength_max else latents
         else:
-            latents = latents.to(device)
-            latents = latents * self.scheduler.init_noise_sigma
+            noise = latents.to(device)
+            latents = noise * self.scheduler.init_noise_sigma
```

Comment:
First the latents are being assigned as `noise` and then they're being assigned as `latents`. I find this exchange somewhat confusing. Could we change that?

Reply:
Hmm ok for me since that's the first time `latents` is seen